apps/worldcapitals/worldcapitals.star

```diff
@@ -0,0 +1,715 @@
+"""
+Applet: WorldCapitals
+Summary: Displays a world capital
+Description: Displays a world capital each day for a specific country.
+Author: Jake Manske
+"""
+
+load("cache.star", "cache")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+load("time.star", "time")
+
+COUNTRIES_REST_ENDPOINT_ALL = "https://restcountries.com/v3.1/all?fields=name,cca3,capital"
+COUNTRIES_REST_ENDPOINT_REGION = "https://restcountries.com/v3.1/region/{0}?fields=name,cca3,capital"
+FONT = "tom-thumb"
+
+# start on Jan 1 2023
+REFERENCE_DATE = time.parse_time("2023-01-01T00:00:00Z")
+
+# other conifg
+HTTP_OK = 200
+CACHE_TIMEOUT = 1  #3600 # 1 hour
```

Comment:
Seems like we could just as well cache for 24h, right?

Reply:
Ah yes, it is a mistake to have the TTL set to 1. You can fix (or I can) with a new PR.

Reply:
I'll fix it. Actually didn't register that it was just 1s before merging. =)